README.md

```diff
@@ -479,6 +481,10 @@ You're already using Nextcloud and want to switch to docker? Great! Here are som
   ```php
   'datadirectory' => '/var/www/html/data',
   ```
+  5. Make sure your Nextcloud lives at the web root. Look for lines like
+  ```php
+  'overwrite.cli.url' => 'https://localhost/nextcloud',
+  ```
```

Comment:
This seems to be unrelated to this PR

Reply:
What I meant is if the user is migrating an existing instance to Docker, and the old one was not at the web root, the installation script would remove the files. So I guess it is related.

Reply:
Please correct me if I'm wrong.

Reply:
IMHO migrating an existing instance to Docker is a different topic.

Reply:
I see your point. I created this as a whole just because my files were deleted, and I found those two places to take a note at. Please feel free to separate the changes if you feel that is better.